### PR TITLE
Add a README file for setting up this repo on Local Machine

### PR DESCRIPTION
**Pull request by NancyNegi23**
## Description
This is a README file for helping out the beginners with cloning this repository hassle free and start their open source contribution journey

Issue: #150 

Note: I've attached some image links which will work perfectly once you merge this PR to the main branch as I've written the path according to the original repository.

### CODE REVIEW

File: CloneRepo.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Clonig a Repository
+
+When you create a repository on GitHub, it exists as a remote repository. You can clone your repository to create a local copy on your computer and sync between the two locations.
+Steps to follow are:
+
+1. On GitHub, navigate to the main page of the repository.
+
+2. Above the list of files, click  Code.
+
+![code](https://raw.githubusercontent.com/Tejas1510/Athavani/master/assets/images%20CloneRepo/code-button.png)
```
Comment (sanjay270899) [inline CloneRepo.md:11]: We can use relative path here. For example: `assets/images%20CloneRepo/code-button.png`.

**Comment by sanjay270899** [comment]
Main intention of that issue was to show how to set-up client and server folder, not just cloning 😅.

**Comment by ChitvanRamani22** [comment]
@NancyNegi23 do you want to improve this further?

**Comment by NancyNegi23** [comment]
No, you can assign this to someone else.

**Pull request closed without merging**